Resources/public/js/alloyeditor/buttons/embed.jsx

```diff
@@ -51,6 +63,23 @@ YUI.add('ez-alloyeditor-button-embed', function (Y) {
         },
 
         /**
+         * Checks if the command is disabled in the current selection.
+         *
+         * @method isDisabled
+         * @return {Boolean} True if the command is disabled, false otherwise.
+         */
+        isDisabled: function () {
+            var path = this.props.editor.get('nativeEditor').elementPath();
+
+            // http://docs.ckeditor.com/#!/api/CKEDITOR.dom.elementPath
+            // There is also isContextFor( tag ), so if there is a way to specify where embeds
+            // are valid that would potentially be cleaner
+            this.state.isDisabled = path && path.contains('table', true) !== null;
```

Comment:
Ok, then I should not use state at all, this is already called on render(), no point causing recursive rendering.

What I was trying to avoid was `isDisabled()` having to be called two times by design, from `getStateClasses()` and `render()`.

Input on how I can avoid welcome, but for next commit I'll get rid of this.